build/pr-pipeline.yml

```
@@ -167,7 +167,7 @@ stages:
       resourceGroup: $(ResourceGroupName)
       sqlServerName: $(DeploymentEnvironmentName)
       schemaAutomaticUpdatesEnabled: 'auto'
-      sqlServerAdminPassword: $[stageDependencies.UpdateRandom.Password.outputs['SetVariablesFromRandomString.password']]
+      userAssignedManagedIdentityName: $(appServicePlanName)
```

I'd stick with `DeploymentEnvironmentName` over the app plan name
```suggestion
      userAssignedManagedIdentityName: $(DeploymentEnvironmentName)
```